client/src/components/AboutPage/AboutHeroSection/AboutHeroSection.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FunctionComponent, MutableRefObject } from 'react';
+import { Link } from 'react-router-dom';
+
+// Components
+import Button from 'components/Button/Button';
+
+// Assets
+import { Wave } from 'assets/images/svg';
+import heroImg1 from 'assets/images/dashboard-landing.png';
+import heroImg2 from 'assets/images/search-landing.png';
+
+// Utils
+import { handleScrollTo } from 'utils/helpers';
+
+// Styles
+import { Container, Flex } from 'styles/global';
+import * as S from '../AboutPage.styles';
+
+interface AboutHeroSectionProps {
+  scrollToRef: MutableRefObject<HTMLElement | null>;
+}
+
+const AboutHeroSection: FunctionComponent<AboutHeroSectionProps> = ({ scrollToRef }) => (
+  <S.AboutHeroSection>
+    <Wave reversed={false} />
```

Comment:
if you mark `reserved` prop as optional and provide default value, it won't be needed to pass `reversed={false}` everywhere